examples/jobs-service.yaml

```
@@ -1,8 +1,9 @@
 apiVersion: app.kiegroup.org/v1alpha1
-kind: KogitoJobsService
+kind: KogitoSupportingService
 metadata:
   name: kogito-jobs-service
```

```suggestion
  name: jobs-service
```

as a matter of harmonizing examples